charmcraft/commands/store/client.py

```diff
@@ -164,8 +159,14 @@ def _storage_push(monitor):
 class Client:
     """Lightweight layer above _AuthHolder to present a more network oriented interface."""
 
-    def __init__(self):
+    # default production URLs
```

Comment:
Also, you may find it nicer to set the parameters with their respective default rather than write as a class/instance var.  Then when creating a client, the parameters are optional (nice for testing etc).

Reply:
As discussed, we'll defer using docstrings for sphinx-generated documentation for a future when charmcraft is a library (we're not generating docs from docstrings as it's only a CLI tool).